languages/javascript/exercises/concept/README.md

```diff
@@ -0,0 +1,60 @@
+# JavaScript concept exercises
+
+The concept exercises in the JavaScript track are a work-in-progress and are based on this [list of concepts][reference-shared]. Important _types_ of concepts to target are things that only exist in object-oriented programming (for people coming from non-oop languages), functions as a first class citizen (for people coming from non-functional languages), and JavaScript specifics:
```

Comment:
We're treating "Concept Exercises" and "Concepts" as _proper nouns_ within the context of Exercism, so they should be capitalized.

Reply:
Oki!